docker/taboot/Dockerfile

```
@@ -0,0 +1,30 @@
+    && apt-get install -y --no-install-recommends \
+        build-essential \
+        curl \
+        git \
+        libffi-dev \
+        libssl-dev \
+        openssh-client \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY . .
+
+RUN pip install --upgrade pip \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)